spec/system/eligibility_spec.rb

```diff
@@ -7,12 +7,26 @@
     then_i_see_the_start_page
   end
 
+  it 'completing the eligibility check' do
+    when_i_visit_the_start_page
+    and_i_press_continue
+    then_i_see_the_eligible_page
+  end
```

Comment:
Can you merge these two specs together?

This spec should be just one seamless run-through of the "happy path."